lib/pages/chat/chat.dart

```diff
@@ -354,6 +352,15 @@ class ChatController extends State<Chat>
     });
   }
 
+  @override
+  void didUpdateWidget(covariant Chat oldWidget) {
+    super.didUpdateWidget(oldWidget);
+
+    if (oldWidget.eventId != widget.eventId && widget.eventId != null) {
```

Comment:
This condition is not correct, as the result when you scroll in server side search, then jump to pinned message, it has an error. remove `widget.eventId != null`